docs/articles/samples/IntroFirstCallColumn.md

```diff
@@ -0,0 +1,58 @@
+---
+uid: BenchmarkDotNet.Samples.IntroFirstCallColumn
+---
+
+## Sample: IntroFirstCallColumn
+
+If you want to have the execution time of the first call to be displayed in a dedicated column, please use `FirstCallColumn`.
+
+### Usage
+
+```cs
+// attribute
+[FirstCallColumn]
+public class MyBenchmarkClass
```

Comment:
The benchmark body is missed.

Reply:
@AndreyAkinshin the body will be included below by `[!code-csharp[IntroColdStart.cs](../../../samples/BenchmarkDotNet.Samples/IntroFirstCallColumn.cs)]`